packages/manager/apps/dedicated/client/app/ip/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -148,65 +150,104 @@ angular.module('Module.ip.controllers').controller(
       return 'USA';
     }
 
+    loadPrivateCloudIpOffers(serviceName) {
```

Comment:
**Suggestion**: May be we can move this to the service... :)

Reply:
i can't :/ i need the selectedService of this.model to be set, i need to wait for the user to select a model